node/tests/GlideClusterClient.test.ts

```diff
@@ -34,7 +34,7 @@ import {
     SlotKeyTypes,
     SortOrder,
 } from "..";
-import { ValkeyCluster } from "../../utils/TestUtils.js";
+import { ValkeyCluster } from "../../utils/TestUtils";
```

Comment:
What's the benifit of this?
And if this is a better way to import `TestUtils`, apart from this file, we still have 2 more files importing `TestUtils` with the `.js` extension: `GlideClient.test.ts` and `ScanTest.test.ts`. Should we change those as well?

Reply:
The benefit is mostly to the IDE, to tell the IDE to look for a ts file if it exist.